src/index.js

```diff
@@ -9,8 +9,11 @@ const logger = getLogger()
 logger.info(`${packageJson.name} started`)
 const startDate = new Date()
 try {
-  const { body: { key } } = await apiRoot.get()
-    .execute()
+  const {
+    body: { key },
+  } = await apiRoot.get().execute()
+
+  logger.info(key)
```

Comment:
Is this log useful without any comment?

Reply:
This is not useful. It is here so that prettier won't complain about unused `key` variable. The `key` variable is there so that we have a proof and an example how to use the apiRoot client. This code will be updated once we actually work on the implementation.